apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2.java

```diff
@@ -273,6 +279,7 @@ private static ExplanationOfBenefit transformClaim(
                       TransformerUtilsV2.createExtensionCoding(
                           eob, CcwCodebookVariable.CARR_LINE_MTUS_CNT, code)));
 
+      String drugCode = drugCodeProvider.retrieveFDADrugCodeDisplay(line.getNationalDrugCode());
```

Comment:
I think it makes sense to inline this (here and below) in the method call since that is more consistent with the other ones and avoids backwards referencing by 50 lines.